docs/howto/local-checkouts.md

```diff
@@ -39,10 +40,72 @@ storage, resulting in cost savings.
 
 <iframe width="420" height="315" src="https://www.youtube.com/embed/afgQnmesLZM"></iframe>
 
-## **lakectl local**: The way to work with lakeFS data locally  
+## **lakeFS Mount**: Efficiently expose lakeFS Data as a local directory
+
+⚠️ lakeFS Mount is currently in preview. There is no installation required, please [contact us](https://info.lakefs.io/thanks-lakefs-mounts) to get access.
+{: .note }
+
+#### Prerequisites:
+
+- A working lakeFS Server running either [lakeFS Enterprise](../enterprise) or [lakeFS Cloud](../cloud)
+- You’ve installed the [`lakectl`](../reference/cli.html) command line utility: this is the official lakeFS command line interface, on top of which lakeFS Mount is built.
+- lakectl is configured properly to access your lakeFS server as detailed in the configuration instructions
+
+### Mounting a lakeFS reference as a local directory
+
+lakeFS Mount works by exposing a virtual mountpoint on the host computer. 
+
+This "acts" as a local directory, allowing applications to read write and interact with data as it is all local to the machine, while lakeFS Mount optimizes this behind the scenes by lazily fetching data as requested, caching accessed objects and efficiently managing metadata to ensure best in class performance. [Read more about how lakeFS Mount optimizes performance](../reference/mount.html)
+
+Mounting a reference is a single command:
+
+```bash
+everest mount lakefs://example-repo/example-branch/path/to/data/ ./my_local_dir
+```
+
+Once executed, the `my_local_dir` directory should appear to have the contents of the remote path we provided. We can verify this:
+
+```bash
+ls -l ./my_local_dir/
+```
+
+Which should return the listing of the mounted path.
+
+
+### Reading from a mount
+
+Reading from a lakeFS Mount requires no special tools, integrations or SDKs! Simply point your code to the directory and read from it as if it was in fact local:
+
+
+```python
+#!/usr/bin/env python
+import glob
+
+for image_path in glob.glob('./my_local_dir/*.png'):
+    with open(image_path, 'rb') as f:
+        process(f)
+
+```
+
+### Unmounting
+
+When done, simply run:
+
+```bash
+everest umount ./my_local_directory
+```
+
+This will unmount the lakeFS Mount, cleaning up background tasks
+
+### Learn more about lakeFS Mount
```

Comment:
Suggestion: Move this to the head of this section and use a `{: .note }` instead of a sub-section with a title.